### PR TITLE
Add gm17 verifier to stdlib

### PR DESCRIPTION
**Pull request by Schaeff**

### CODE REVIEW

File: zokrates_stdlib/tests/tests/snark/gm17.zok [hunk 1]
```diff
@@ -26,34 +26,27 @@
...
-// 				"values": [' >> gm17.json
-
-// cat proof.json >> gm17.json
-
-// echo ', ' >> gm17.json
-
-// cat verification.key >> gm17.json
-
-// echo '   
+// cat > gm17.json << EOT
```
Comment (dark64) [inline zokrates_stdlib/tests/tests/snark/gm17.zok:29]: @Schaeff we can use templating here

File: zokrates_stdlib/tests/tests/snark/gm17.zok [hunk 2]
```diff
@@ -26,34 +26,27 @@
...
 // ```sh
-// rm gm17.json
-// touch gm17.json
-
-// echo '{
-// 	"entry_point": "./tests/tests/snark/gm17.zok",
-// 	"curves": ["Bw6_761"],
-// 	"tests": [
-// 		{
-// 			"input": {
```
Comment (dark64) [inline zokrates_stdlib/tests/tests/snark/gm17.zok]: `abi` flag was missing

File: zokrates_stdlib/tests/tests/snark/gm17.zok (hunk 1 above)
Comment (Schaeff) [reply zokrates_stdlib/tests/tests/snark/gm17.zok:29]: I'll trust you on this!

File: zokrates_stdlib/tests/tests/snark/gm17.zok (hunk 2 above)
Comment (Schaeff) [reply zokrates_stdlib/tests/tests/snark/gm17.zok]: nice, feel free to merge if you're happy with it

**Pull request merged**